std/exception.d

```diff
@@ -2145,3 +2145,58 @@ pure nothrow @safe unittest
 
     auto infSlice = infinite[0 .. $]; // this would throw otherwise
 }
+
+
+/**
+ * Convenience mixin for trivially sub-classing exceptions
+ *
+ * Even trivially sub-classing an exception involves writing boilerplate code
+ * for the constructor to correctly pass in the source file and line number the
+ * exception was thrown from. This mixin makes it easier.
```

Comment:
I'd suggest changing this to something like "`exception was thrown from or to make it work with $(LREF enforce)`" - since making it work with `enforce` is a key reason to use the signatures that get mixed in with this template mixin rather than another constructor that gets the file and line number right but does something like put them in the reverse order from what `enforce` expects.